app/views/support_interface/data_exports/view_export_information.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, @data_export[:name] %>
+
+<% content_for :before_content do %>
+  <%= render BreadcrumbComponent.new(items: [{
+    text: 'Performance',
+    path: support_interface_performance_path,
+  }, {
+    text: 'Data directory',
+    path: support_interface_data_directory_path,
+  }, {
+    text: @data_export.fetch(:name),
+  }]) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body"><%= @data_export.fetch(:description) %></p>
+
+    <%= govuk_button_to 'Generate new export', support_interface_data_exports_path(export_type_id: params[:data_export_id]), class: 'govuk-button--secondary' %>
+
+    <p class="govuk-body"><%= govuk_link_to 'View download history', support_interface_view_download_history_path(data_export_id: params[:data_export_id]) %></p>
```

Comment:
Is it download history or just "History"? (We don't record what's actually been downloaded)

Reply:
Good point - I suppose it's more so a history of when an export has been generated. The more detailed log is within the individual export page.

Originally I did have it as just "history" but i didn't know if this was explicit enough as to what type of history it was referring to. Perhaps overthinking, so happy to revert it back!